packages/esm-patient-programs-app/src/programs/programs-action-menu.component.tsx

```diff
@@ -15,8 +15,13 @@ export const ProgramsActionMenu = ({ patientUuid, programEnrollmentId }: Program
   const isTablet = useLayoutType() === 'tablet';
 
   const launchEditProgramsForm = useCallback(
-    () => launchPatientWorkspace('programs-form-workspace', { programEnrollmentId }),
-    [programEnrollmentId],
+    () =>
+      launchPatientWorkspace('programs-form-workspace', {
+        workspaceTitle: t('editProgramEnrollment', 'Edit program enrollment'),
+        programEnrollmentId,
+        formContext: 'editing',
```

Comment:
We aren't using this property to explicitly determine the form mode; instead, [we're inferring that from the `programEnrollmentId`](https://github.com/openmrs/openmrs-esm-patient-chart/pull/2226/files#diff-57708a7f6097ba5e306b050c9b41170f62976d76b03676fa010e2e826729d212R64); @denniskigen is this by design?

Reply:
Not at all, I must've accidentally introduced this after looking at the Conditions edit flow. Fixed.